.github/DEVELOPMENT.md

```diff
@@ -150,7 +150,9 @@ Disable the following inspections:
 
 - ``Java | Performance | Call to 'Arrays.asList()' with too few arguments``,
 - ``Java | Abstraction issues | 'Optional' used as field or parameter type``,
-- ``Java | Data flow | Boolean method is always inverted``.
+- ``Java | Data flow | Boolean method is always inverted``,
+- ``JVM languages | Unstable API usage``
+  - or, at least, remove ``com.google.common.annotations.Beta``, as we use a lot of beta APIs from Guava.
```

Comment:
We should be specific here. 
let's not suggest disable ``JVM languages | Unstable API usage``